enclave/ecalls.cpp

```diff
@@ -211,6 +211,11 @@ int enclave_add_client_key(
     return add_client_key(data, data_len, signature, sig_len);
 }
 
+int enclave_sync_client_key() {
```

Comment:
See comment for `c_api.h`. Not sure if we need to expose this function to the client.